xml-ws-tck/run-tck.sh

```diff
@@ -8,8 +8,8 @@ fi
 export TS_HOME=`pwd`/xml-ws-tck
 export PATH=${PATH}:${TS_HOME}/bin
 
-rm jakarta-xml-ws-tck-4.0.0.zip
-wget https://download.eclipse.org/jakartaee/xml-web-services/4.0/jakarta-xml-ws-tck-4.0.0.zip
+#rm jakarta-xml-ws-tck-4.0.0.zip
```

Comment:
Should we be removing this cleanup?
```suggestion
rm jakarta-xml-ws-tck-4.0.0.zip
```

Reply:
Well the tcks are immutable and quite slow to download, making interactive use painful. But yes, on rare occasions we may just kill the line when necessary